zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -74,22 +76,45 @@ const char * zstdVersion() { return ZSTD_VERSION_STRING; }
 ZEXTERN const char * ZEXPORT z_zlibVersion OF((void)) { return zlibVersion();  }
 
 
+void* ZWRAP_allocFunction(void* opaque, size_t size)
```

Comment:
This function prototype is not `static`, hence it is accessible from outside.
Is it intended ?